Program.cs

```diff
@@ -111,6 +111,19 @@ static void CategoryOfHeigh()
         {
             Console.Write("Podaj swoj wzrost (w cm) a powiem Ci kim jesteś: ");
             double heigh = double.Parse(Console.ReadLine());
+
+            if (heigh < 70)
+                Console.WriteLine("Jesteś gnomem. Say hello to Gollum!\n");
+            if (heigh >= 70 && heigh < 100)
+                Console.WriteLine("Jesteś niziołkiem. Say hello to Frodo!\n");
+            if (heigh >= 100 && heigh < 150)
+                Console.WriteLine("Jesteś krasnoludem. Say hallo to Gimli!\n");
+            if (heigh >= 150 && heigh < 200)
+                Console.WriteLine("Jesteś elfem. Say hello to Legolas!\n");
+            if (heigh >= 200)
+                Console.WriteLine("Jesteś czarodziejem. Say hallo to Gandalf!\n");
```

Comment:
Now it looks better! The code is more visibly and you exit from conditions asap.